frontend/src/MessageBoard/components/ActivityTimeline.js

```diff
@@ -36,7 +36,9 @@ export default class ActivityTimeline extends React.Component {
 						//query into messages and allTheThings.
 						if (messages.length === 0) {
 							messages = findMessagesByTeam;
-							allTheThings = allTheThings.concat(messages);
+							for (var message of messages) {
```

Comment:
Unless you have a very convincing argument for using `var` here, please update it to `let`.